ui/js/plugins/userfas/userfas.js

```diff
@@ -60,6 +60,17 @@ define([
                 }, {
                     name: 'fasstatusnote',
                     flags: ['w_if_no_aci']
+                }, {
+                    $type: 'radio',
+                    name: 'faspronoun',
+                    flags: ['w_if_no_aci'],
+                    default_value: '',
+                    options: [
+                        { label: '@i18n:userfas.pronoun_na', value: '' },
```

Comment:
shouldn't the value be "n/a", since that's what is used in the i18n_messages variable ?

Reply:
Oups, no I misundertood the code, ignore my remark :/

Reply:
Yeah, empty value is internally translated to no value set.